macros/src/lib.rs

```diff
@@ -199,10 +206,8 @@ fn expand_tracing_init(attribute_args: &AttributeArgs) -> Tokens {
       let __internal_event_filter = {
         use ::test_log::tracing_subscriber::fmt::format::FmtSpan;
 
-        match ::std::env::var_os("RUST_LOG_SPAN_EVENTS") {
+        match ::core::option_env!("RUST_LOG_SPAN_EVENTS") {
```

Comment:
We should preserve the runtime behavior, at least for `std` contexts. Can you introduce an `std` feature and make this conditional?